.github/workflows/latest_deps.yml

```diff
@@ -84,6 +84,12 @@ jobs:
         if: ${{ matrix.postgres-version }}
         timeout-minutes: 2
         run: until pg_isready -h localhost; do sleep 1; done
+
+      # We nuke the local copy, as we've installed synapse into the virtualenv
+      # (rather than use an editable install, which we no longer support). If we
+      # don't do this then python can't find the native lib.
+      - run: rm -rf synapse/
+
```

Comment:
Aha, I guess this was an oversight from before then. Presumably the local copy wins as far as the PYTHONPATH is concerned?